### DISCUSSION TITLE
After 'brew install openjdk' (15.0.1), 'java -version' show 'openjdk version "16" 2021-03-16'

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#996] [Everyday usage] [answered]
I installed OpenJDK use `brew install openjdk` command. Then I checked my JDK version using `java -version`, it shows:
```bash
openjdk version "16" 2021-03-16
OpenJDK Runtime Environment (build 16+14)
OpenJDK 64-Bit Server VM (build 16+14, mixed mode)
```
The JDK information in VSCode is also confusing. It shows `JavaSE-15` in status bar, and shows `version 16` in `Configure Java Runtime` setting page.

Can you explain it? Thanks~

**Comment by Bo98** [comment, marked as answer]
Are you running Apple Silicon/M1? We currently install a Java 16 preview (though still versioned as 15 in `brew` because that's what we use on Intel) since Java 15 is not supported on ARM.

**Reply by ghost** [reply]
Yes, I'm using Silicon Mac. I see and thank you for your reply.

**Reply by SMillerDev** [reply]
The install will have given you a heads up about this too.
